app/helpers/application_helper.rb

```diff
@@ -53,4 +53,49 @@ def render_index_grid(grid)
     no_records = grid.current_page_records.length < 1
     render "admin/common/grid", no_records: no_records, grid: grid
   end
+
```

Comment:
Yeah I agree, we should move this. Seems like overkill to put in its own class, but I would be fine with either way.